src/routes/browse.js

```diff
@@ -153,14 +155,28 @@ export default function Browse() {
     );
   }
 
+  const onBidChanged = (e) => {
+    if (e.target.value == "") {
+      setBidIsValid(false);
+      return
+    }
+    if (!isNaN(+e.target.value)) {
+      if (Number(+e.target.value) < 0) {
+        setBidIsValid(false);
+        return;
+      }
+      setBidIsValid(true);
+    }
+  }
+
   // Map-Related Constants
   const [selectedLocation, setSelectedLocation] = useState({
     lat: 48.4284,
     lng: -123.3656,
   });
   const [radius, setRadius] = useState(0);
   const [search_value, setSearchValue] = useState(null);
-  const apiKey = 'AIzaSyBN9FNice6SVThI5Yo_MmQS9Or-votMad8';
+  const apiKey = 'AIzaSyBeu0giaCGz9vMeTn1HBRtTRlWQuVZwpZs';
```

Comment:
I don't think we should be committing the apiKey's as this new one is for localHost debugging only

@DH3773 would that be correct?